src/wooting-usb.c

```diff
@@ -46,8 +44,50 @@ static uint16_t getCrc16ccitt(const uint8_t* buffer, uint16_t size)
 	return crc;
 }
 
-void wooting_usb_disconnect(bool trigger_cb) {
+#pragma region Set Meta
+
+void reset_meta() {
 	wooting_usb_meta.connected = false;
+
+	wooting_usb_meta.model = "N/A";
+	wooting_usb_meta.device_type = -1;
+	wooting_usb_meta.keycode_limit = 0;
+	wooting_usb_meta.max_rows = 0;
+	wooting_usb_meta.max_columns = 0;
+	wooting_usb_meta.keycode_limit = 0;
+}
+
+void set_meta_wooting_one() {
+	wooting_usb_meta.model = "One";
```

Comment:
Prefer "Wooting one" and "Wooting two" for models

Reply:
Slight side note, a nice consequence of the numbering I did for the `WOOTING_DEVICE_TYPE` enum is that the type corresponding to the Wooting One is no 1 and 2 for Wooting Two. So that can also be used to determine what keyboard it is.

Reply:
I like to try and avoid using strings to indicate something relatively constant where possible. Do you think it'd be worthwhile to change the model to a model enum?